### PR TITLE
chore(wiki): add third-party content disclaimer

### PR DESCRIPTION
**Pull request by DannyS03**
## Context

Creates a general content disclaimer hosted in the repo and adds caution admonitions across the Wiki to refer to the third-party content disclaimer.

### CODE REVIEW

File: docs/develop/alchemy.md [hunk 1]
```diff
@@ -13,6 +13,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/alchemy.md:18]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/chainide.md [hunk 2]
```diff
@@ -16,6 +16,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/chainide.md:21]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/chainstack.md [hunk 3]
```diff
@@ -14,6 +14,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/chainstack.md:19]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/crusthelpers.md [hunk 4]
```diff
@@ -13,6 +13,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/crusthelpers.md:18]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/dagger.md [hunk 5]
```diff
@@ -13,6 +13,14 @@ image: https://wiki.polygon.technology/img/polygon-logo.png
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/dagger.md:18]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/data/covalent.md [hunk 6]
```diff
@@ -13,16 +13,24 @@ keywords:
   - index
   - indexing
   - query
-image: https://matic.network/banners/matic-network-16x9.png 
+image: https://matic.network/banners/matic-network-16x9.png
+---
+
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/data/covalent.md:21]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/data/graph-data.md [hunk 7]
```diff
@@ -16,6 +16,14 @@ keywords:
 image: https://matic.network/banners/matic-network-16x9.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/data/graph-data.md:21]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/data/graph-entities.md [hunk 8]
```diff
@@ -16,32 +16,40 @@ keywords:
 image: https://matic.network/banners/matic-network-16x9.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/data/graph-entities.md:21]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/data/graph-queries.md [hunk 9]
```diff
@@ -16,6 +16,14 @@ keywords:
 image: https://matic.network/banners/matic-network-16x9.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/data/graph-queries.md:21]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/data/the-graph.md [hunk 10]
```diff
@@ -6,18 +6,26 @@ description: Learn how to set up a hosted project with The Graph and Polygon.
...
-slug: graph 
+slug: graph
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-The Graph, a decentralized protocol for indexing and querying chain data, supports the Matic chain. Data defined through subgraphs is easy to query and explore. Subgraphs can be created locally, or use a free hosted explorer for indexing and data display. 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/data/the-graph.md:22]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/developer-resources.md [hunk 11]
```diff
@@ -16,6 +16,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/developer-resources.md:21]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/filecoinhelpers.md [hunk 12]
```diff
@@ -16,6 +16,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/filecoinhelpers.md:21]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/getblock.md [hunk 13]
```diff
@@ -14,9 +14,17 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/getblock.md:19]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/metamask/config-polygon-on-metamask.md [hunk 14]
```diff
@@ -14,6 +14,14 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/metamask/config-polygon-on-metamask.md:19]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/metamask/config-polygon-on-wallet-link.md [hunk 15]
```diff
@@ -7,10 +7,18 @@ keywords:
...
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
+
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/metamask/config-polygon-on-wallet-link.md:17]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/metamask/custom-tokens.md [hunk 16]
```diff
@@ -11,6 +11,14 @@ image: https://wiki.polygon.technology/img/polygon-logo.png
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/metamask/custom-tokens.md:16]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/metamask/multiple-accounts.md [hunk 17]
```diff
@@ -7,8 +7,17 @@ keywords:
   - polygon
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
+
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/metamask/multiple-accounts.md:15]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/metamask/overview.md [hunk 18]
```diff
@@ -11,6 +11,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/metamask/overview.md:16]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/metamask/tutorial-metamask.md [hunk 19]
```diff
@@ -8,8 +8,17 @@ keywords:
   - matic
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
+
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/metamask/tutorial-metamask.md:16]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/nftstorage.md [hunk 20]
```diff
@@ -12,6 +12,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/nftstorage.md:17]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/oracles/api3.md [hunk 21]
```diff
@@ -14,6 +14,14 @@ image: https://wiki.polygon.technology/img/polygon-wiki.png
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/oracles/api3.md:19]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/oracles/bandchain.md [hunk 22]
```diff
@@ -12,8 +12,17 @@ keywords:
   - band protocol
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
+
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/oracles/bandchain.md:20]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/oracles/bandstandarddataset.md [hunk 23]
```diff
@@ -13,8 +13,17 @@ keywords:
   - band protocol
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
+
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/oracles/bandstandarddataset.md:21]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/oracles/chainlink.md [hunk 24]
```diff
@@ -15,6 +15,14 @@ image: https://wiki.polygon.technology/img/polygon-logo.png
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/oracles/chainlink.md:20]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/oracles/diadataoracles.md [hunk 25]
```diff
@@ -9,6 +9,15 @@ keywords:
   - oracle
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
+
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/oracles/diadataoracles.md:17]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/oracles/getting-started.md [hunk 26]
```diff
@@ -21,6 +21,12 @@ Inbound oracles bring off-chain or real-world data to the blockchain, whereas th
 
 ## Blockchain Oracles
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/oracles/getting-started.md:26]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/oracles/optimisticoracle.md [hunk 27]
```diff
@@ -12,6 +12,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/oracles/optimisticoracle.md:17]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/oracles/razor.md [hunk 28]
```diff
@@ -11,6 +11,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/oracles/razor.md:16]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/oracles/tellor.md [hunk 29]
```diff
@@ -11,6 +11,14 @@ source: Tellor Docs
 sourceUrl: https://docs.tellor.io/tellor/
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/oracles/tellor.md:16]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/quicknode.md [hunk 30]
```diff
@@ -16,6 +16,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/quicknode.md:21]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/tools/graph.md [hunk 31]
```diff
@@ -5,12 +5,20 @@ description: Learn how to set up a hosted project with The Graph and Polygon.
...
-image: https://matic.network/banners/matic-network-16x9.png 
+image: https://matic.network/banners/matic-network-16x9.png
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-The Graph, a decentralized protocol for indexing and querying chain data, supports the Polygon chain. Data defined through subgraphs is easy to query and explore. Subgraphs can be created locally, or use a free hosted explorer for indexing and data display. 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/tools/graph.md:15]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/wallets/plaid-wallet-onboard.md [hunk 32]
```diff
@@ -15,6 +15,14 @@ image: https://wiki.polygon.technology/img/polygon-logo.png
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/wallets/plaid-wallet-onboard.md:20]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/wallets/portis.md [hunk 33]
```diff
@@ -11,6 +11,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/wallets/portis.md:16]:
```suggestion
Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/wallets/slashauth.md [hunk 34]
```diff
@@ -12,6 +12,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/wallets/slashauth.md:17]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/wallets/torus.md [hunk 35]
```diff
@@ -12,6 +12,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/wallets/torus.md:17]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/wallets/venly/create_wallet.md [hunk 36]
```diff
@@ -13,6 +13,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/wallets/venly/create_wallet.md:18]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/wallets/venly/custom_tokens.md [hunk 37]
```diff
@@ -13,6 +13,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/wallets/venly/custom_tokens.md:18]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/wallets/venly/intro_arkane.md [hunk 38]
```diff
@@ -14,6 +14,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/wallets/venly/intro_arkane.md:19]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/develop/wallets/venly/network.md [hunk 39]
```diff
@@ -14,6 +14,14 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-logo.png
 ---
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/develop/wallets/venly/network.md:19]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

File: docs/operate/archive-node-nodereal.md [hunk 40]
```diff
@@ -15,6 +15,14 @@ image: https://wiki.polygon.technology/img/polygon-logo.png
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
+:::caution Content disclaimer
+
+Please view the third-party content disclaimer [[ here ]](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```
Comment (ashucoder9) [inline docs/operate/archive-node-nodereal.md:20]:
```suggestion
Please view the third-party content disclaimer [<ins>here</ins>](https://github.com/maticnetwork/matic-docs/blob/master/CONTENT_DISCLAIMER.md).
```

**Review by ashucoder9 — APPROVED** [review]
Left some suggestions, thanks @DannyS03

**Comment by cerberushades** [comment]
@DannyS03 I sent a PR to make some adjustments on NodeReal tutorial.

**Pull request merged**